boards/microchip/sam/sam_e54_xpro/sam_e54_xpro.dts

```diff
@@ -57,6 +58,14 @@
 	};
 };
 
+&rtc {
+	compatible = "microchip,rtc-g1";
+	prescaler = <1024>;
+	alarms-count = <2>;
+	cal-constant = <1048576>;  /* (8192 * 128 = 1048576) */
```

Comment:
Same with this constant. User should not know about this if it never changes. This means that is should be defined at dts/arm/microchip/sam/sam_d5x_e5x/common/samd5xe5x.dtsi.